pages/common/eim.md

```diff
@@ -0,0 +1,36 @@
+# eim
+
+> Install and manage ESP-IDF.
+> More information: <https://docs.espressif.com/projects/idf-im-ui/en/latest/cli_commands.html>.
+
+- Install default (latest) ESP-IDF version on default location (C:\esp on Windows and ~/.espressif on POSIX systems):
+
+`eim install`
+
+- Install a specific ESP-IDF version non-interactively (default headless mode):
```

Comment:
```suggestion
- Install a specific ESP-IDF version:
```
Not sure if this is necessary. Share your opinion if you think it's needed.

Reply:
You are right. It's not needed. I just want to distinguish it from the wizard. But that's probably self-explanatory.